src/styles/components/quota/styles.less

```diff
@@ -40,6 +40,5 @@
   /* These values serve different purposes */
   /* stylelint-disable declaration-block-no-duplicate-properties */
   min-height: 300px;
-  min-height: -webkit-fill-available;
```

Comment:
that made the table below the quota-cards invisible in browser that support this property. it may have worked before the cards got implemented or if you have a very "high" browser window.